### DISCUSSION TITLE
(Suggestion)Legit AA

### DISCUSSION THREAD

**Discussion by Ragdoll117** [Fedoraware/Fedoraware#1037] [Ideas]
Legit AA like rijin

**Comment by Ragdoll117** [comment]
![image](https://github.com/Fedoraware/Fedoraware/assets/102504389/d7d5a40b-0ab5-4a49-9b63-646dc61c6d74)

**Reply by canedoly** [reply]
what's the issue here

**Reply by Ragdoll117** [reply]
I enabled bot_mimic 1, the bot is mimicking my legit aa and it's flicking between real and fake (I turned off my resolver and that's what it looks like)

**Reply by canedoly** [reply]
who would have guessed. bots cannot choke packets therefore they can't fake angles

**Comment by Ragdoll117** [comment]
> ![image](https://user-images.githubusercontent.com/102504389/266024919-d7d5a40b-0ab5-4a49-9b63-646dc61c6d74.png)
